### DISCUSSION TITLE
Langzeit Zuverlässigkeit? Kompakte Bauform?

### DISCUSSION THREAD

**Discussion by iamthe1st** [frickelzeugs/FingerprintDoorbell#101] [Q&A]
Hi, ich habe einen normalen Gira-Taster mit einer Unterputzdose als Klingel-Taster.
Das möchte ich gerne gegen eine Klingel mit Fingerabdruck-Scanner tauschen.
Wie sind eure Langzeiterfahrungen mit dem Fingerabdrucksensor?

Ist es denkbar alle Teile inkl. Klingel-Relais in einer Unterputzdose unterzubringen?

Ist es sinnvoller Sensor und Klingeltaster getrennt einzubauen zwecks Zuverlässigkeit?

**Comment by tilly001** [comment]
Hi,

Ich betreibe die Kombination R503 mit ESP schon fast 2 Jahre ohne Probleme. 
Wenn es eine tiefe UP Dose ist, sollte es auch ohne Probleme passen. Ich habe es in meinen Briefkasten eingebaut. 

Die Software von frickekzeugs habe ich seit ca. 3 Monaten drauf.

Gruß Stefan

**Comment by iamthe1st** [comment]
Danke für die Rückmeldung!
Um den GPIO für die Klingel zu ändern, muss ich den Quellcode anpassen - oder geht das nachträglich per MQTT / Webinterface?

Für den D1 Mini gibt es neben einem 24V shield und auch ein Relais-shield -- das verwendet allerdings den "D1" Pin - nicht IO33.

Jetzt brauch ich nur noch ein "smartes" Türschloss -  kann man das direkt mit diesem ESP32 koppen, oder muss das indirekt über HomeAssistant an-gesteuert werden?

**Reply by tilly001** [reply]
Ich habe die Klingel noch separat. 

Scheinbar kann man es auch MQTT ändern. 
Ich kann dir aber nicht sagen wie. 


![Screenshot_20241117_164833_Chrome](https://github.com/user-attachments/assets/7081e338-36ab-4617-a6e9-64c41fe4ac10)

Zum smarten Türschloss kann ich dir Tedee mit einer Bridge empfehlen. Läuft seit Jahren ohne Probleme. Kann in den IoBroker integriert werden. 

[https://www.iobroker.net/#en/adapters/adapterref/iobroker.tedee/README.md](url)

Gruß Stefan

**Reply by PKlhomesmart** [reply]
Hi Stefan, 

ich habe das ganze System auch bei mir installiert, nur die Klingel habe ich nicht extra über GPIO19 angeschlossen. 
Wie hast du das gemacht. Es ist ja nur ein Ausgang und da ist nie ein Signal (High oder 5V oder so) angekommen. 

Zur Info, ich habe die Klingel einfach über MQTT mit einem Zigbee Relais verbunden. Nur möchte ich das Ganze auch separtart anschließen.

**Reply by tilly001** [reply]
Meine Klingel läuft separat und wird über einen weiteren ESP8266 ausgewertet. 

Wenn hier am Fingerprint Sensor kein Finger erkannt wird, wird per MQTT ein Klingeln ausgelöst. Der GPIO 19 ist nicht belegt bei mir.

**Reply by PKlhomesmart** [reply]
Danke!
